modules/markdown/markdown.go

```diff
@@ -542,7 +542,7 @@ func RenderCrossReferenceIssueIndexPattern(rawBytes []byte, urlPrefix string, me
 func renderSha1CurrentPattern(rawBytes []byte, urlPrefix string) []byte {
 	ms := Sha1CurrentPattern.FindAllSubmatch(rawBytes, -1)
 	for _, m := range ms {
-		all := m[0]
+		all := m[1]
```

Comment:
I think we should not call it `all` now that right fully catch only the commitId/sha1.